scripts/linters/pylint_extensions_test.py

```diff
@@ -2165,3 +2165,50 @@ def test_file_overview_at_end_of_file(self):
 
         with self.checker_test_object.assertAddsMessages(message):
             temp_file.close()
+
+
+class IndentMultilineDocstringDefinitionCheckerTests(unittest.TestCase):
```

Comment:
Add more tests. For example,
1.) If docstring is inside a method.
2.) For `Returns` tag.